Fundamentals/Setup/Requirements/index.md

```diff
@@ -50,3 +50,15 @@ You can use Umbraco Cloud to manage the hosting infrastructure. All Umbraco Clou
 * Ability to set file permissions to include create/read/write (or better) for the user that "owns" the Application Pool for your site (NETWORK SERVICE, typically)
 
 *For more information, see the [Host and deploy ASP.NET Core applications](https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/?view=aspnetcore-6.0) article in the Microsoft documentation.*
+
+## Database Account Roles
+
+The database account used in the connection string will need permissions to read and write from tables as well as create schema during installs and upgrades:
+
+* The `db_owner` role has full permissions on the database.
+
+* The `db_datareader` and `db_datawriter` roles will be needed for normal use to use an account with more restricted permissions.
+
+* The `db_ddladmin` role can perform installs and upgrades of the CMS and/or any packages that create database tables.
```

Comment:
```suggestion
* To use an account with more restricted permissions, the `db_datareader` and `db_datawriter` roles will be needed for normal use to read from and write to the database.  The `db_ddladmin` role, which can modify database schema, is required for installs and upgrades of the CMS and/or any packages that create database tables.
```

I suggest the above for this, as overall there are really two options - either you use `db_owner` (which can do everything)., or you use an account with only the necessary roles, which then are listed out.